examples/call_highs_from_python.py

```diff
@@ -1,14 +1,42 @@
-file = "../src/interfaces/highs_lp_solver.py"
+file = "../HiGHS/src/interfaces/highs_lp_solver.py"
 exec(compile(open(file).read(), file, 'exec'))
 
-cc = (1.0, -2.0)
-cl = (0.0, 0.0)
-cu = (10.0, 10.0)
-ru = (2.0, 1.0)
-rl = (0.0, 0.0)
-astart = (0, 2)
-aindex = (0, 1, 0, 1)
-avalue = (1.0, 2.0, 1.0, 3.0)
-rc, cv, cd, rv, rd , cbs, rbs = Highs_call(cc, cl, cu, rl, ru, astart, aindex, avalue)
-
-print(rc, cv, cd, rv, rd, cbs, rbs)
+# Problem is
+#
+# min 8 x_1 + 10 x_2
+#
+# st  2 x_1 +  2 x_2 >= 7
+#     3 x_1 +  4 x_2 >= 12
+#     2 x_1 +    x_2 >= 6
+#
+#       x_1 >= 0 x_2 >= 0
+#
+inf = 1e30
+colcost = (8.0, 10.0)
+collower = (0.0, 0.0)
+colupper = (inf, inf)
+rowlower = (7.0, 12.0, 6.0)
+rowupper = (inf, inf, inf)
+astart = (0, 3)
+aindex = (0, 1, 2, 0, 1, 2)
+avalue = (2.0, 3.0, 2.0, 2.0, 4.0, 1.0)
+
+# Find the continuous solution of the LP
+return_code, col_value, col_dual, row_value, row_dual , col_basis, row_basis = Highs_lpCall(colcost, collower, colupper,
+                                                                                            rowlower, rowupper,
+                                                                                            astart, aindex, avalue)
+
+print return_code, col_value, col_dual, row_value, row_dual, col_basis, row_basis
```

Comment:
These `print` statements will not work with Python 3 (the current and supported version) and should have parentheses, i.e.
```python
print(return_code, col_value, col_dual, row_value, row_dual, col_basis, row_basis)
```